skyportal/handlers/api/annotation.py

```diff
@@ -8,13 +8,32 @@
 
 class AnnotationHandler(BaseHandler):
     @auth_or_token
-    def get(self, annotation_id):
+    def get(self, associated_resource_type, resource_id, annotation_id):
         """
         ---
         description: Retrieve an annotation
         tags:
           - annotations
         parameters:
+          - in: path
+            name: associated_resource_type
+            required: true
+            schema:
+              type: string
+            description: |
+               What underlying data the annotation is on:
+               an "object" (default), or a "spectrum".
+          - in: path
+            name: resource_id
+            required: true
+            schema:
+              type: string
+            description: |
+               The ID of the underlying data.
+               This would be a string for an object ID
+               or an integer for other data types like spectrum.
+               The annotation ID must correspond to an annotation on the
```

Comment:
Can we clarify this?  I think users may have a hard time parsing what this means.